pkg/compute/manifest/manifest.go

```diff
@@ -58,6 +62,32 @@ func (d *Data) ServiceID() (string, Source) {
 	return "", SourceUndefined
 }
 
+// Description yields a Description.
+func (d *Data) Description() (string, Source) {
+	if d.Flag.Description != "" {
+		return d.Flag.Description, SourceFlag
+	}
+
+	if d.File.Description != "" {
+		return d.File.Description, SourceFile
+	}
+
+	return "", SourceUndefined
+}
+
+// Authors yields an Authors.
+func (d *Data) Authors() ([]string, Source) {
+	if len(d.Flag.Authors) > 0 {
+		return d.Flag.Authors, SourceFlag
+	}
+
+	if len(d.File.Authors) > 0 {
+		return d.File.Authors, SourceFile
+	}
```

Comment:
Would it make sense to combine them?

Reply:
They are subtly different. One returns the `Flag` value, the other returns the `File` value.